install_xdproot.sh

```diff
@@ -18,6 +18,10 @@
 # Example script to start and run xdproot program
 set -xeo pipefail
 INTERFACE="enp0s3"
+if [ -f /etc/redhat-release ]; then
+    INTERFACE="eth0"
```

Comment:
You can change it to user-supplied value rather than based on some arbitrary check such as this.